src/components/screens/ExperimentCreator.vue

```diff
@@ -0,0 +1,1348 @@
+<template>
+  <EmailPrompt
+      dialog="You must add an email address to your account to create or edit an experiment. You can do so below, or on the 'Settings' page."
+      :isFirstLoginPrompt="false"
+  />
+  {{ experimentSetUrn }}
+  <DefaultLayout>
+    <div class="mave-experiment-editor">
+      <div v-if="itemStatus != 'NotLoaded'" class="mave-screen-title-bar">
+        <div class="mave-screen-title">Edit experiment {{ this.item.urn }}</div>
+        <div v-if="item" class="mave-screen-title-controls">
+          <Button @click="saveEditContent">Save changes</Button>
+          <Button @click="resetForm" class="p-button-help">Clear</Button>
+          <Button @click="viewItem" class="p-button-warning">Cancel</Button>
+        </div>
+      </div>
+      <div v-else class="mave-screen-title-bar">
+        <div class="mave-screen-title">Create a new experiment</div>
+        <div class="mave-screen-title-controls">
+          <Button @click="validateAndSave">Save</Button>
+          <Button @click="resetForm" class="p-button-help">Clear</Button>
+          <Button @click="backDashboard" class="p-button-warning">Cancel</Button>
+        </div>
+      </div>
+      <div class="mavedb-wizard">
+        <Stepper v-model:activeStep="activeWizardStep">
```

Comment:
I think it would be nice to have another step before the information page where the user can select the experiment set they will add their experiment to, sort of like what we have where users can select which experiment to add their score set to.

Reply:
That's what I considered before too. But experiment set only has an urn and I think it's a bad dropdown menu cause they're all too similar. In order to distinguish them, users still need to find the experiment set first. And not every new experiment belongs to an existing experiment set. Hence I think it's fine if they want to add it to an existing experiment set, then find the experiment set first. Otherwise, just creating a new experiment set for this experiment directly. That's why I didn't add this part.

Reply:
Yeah, that makes sense. Thanks for the explanation